From 9b04353f502d3fa371231742817b07cb0c41f80c Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 8 Feb 2021 11:15:39 +0000
Subject: [PATCH] profiling_host: depend on zlib instead of headers only

There is no zlib_common_headers target for unbundled zlib
and in the end code links against zlib static library anyways.
---
 chrome/browser/profiling_host/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/profiling_host/BUILD.gn b/chrome/browser/profiling_host/BUILD.gn
index a68ac82..1ec9d66 100644
--- a/chrome/browser/profiling_host/BUILD.gn
+++ b/chrome/browser/profiling_host/BUILD.gn
@@ -24,7 +24,7 @@ static_library("profiling_host") {
     "//components/version_info",
     "//content/public/browser",
     "//content/public/common",
-    "//third_party/zlib:zlib_common_headers",
+    "//third_party/zlib:zlib",
 
     # Added to support the dependency on //chrome/browser/metrics
     # TODO(darin): We should instead have a way to express that dependency here.
-- 
2.26.2

